beacon-chain/p2p/testing/p2p.go

```diff
@@ -101,6 +101,7 @@ func (p *TestP2P) ReceiveRPC(topic string, msg proto.Message) {
 
 	n, err := p.Encoding().EncodeWithMaxLength(s, msg)
 	if err != nil {
+		s.Reset()
```

Comment:
Please handle this error from s.Reset(). (Static analysis will not allow the build to complete with unhandled errors)